swedish-oidc-claims-specification.md

```diff
@@ -390,104 +388,156 @@ There are basically two ways for a Relying Party to indicate which claims it wis
+**Description:** A scope that requests a Swedish civic registration number (personnummer) or 
+a Swedish coordination number (samordningsnummer). These identity numbers are often used as 
+primary identities at public Swedish organizations.
+
+Note that an OpenID Provider delivering claims according to this scope MUST NOT deliver both a
+`personalIdentityNumber` and a `coordinationNumber` claim. These are mutually exclusive. However, 
+a person that holds a civic registration number (personnummer) may previously have had a 
+coordination number. This number may then be delivered using the `previousCoordinationNumber` 
+claim (see section [2.1.2.2](#previous-coordination-number)). This claim has to be explicitly
+requested using the `claims` request parameter and is not part of the `naturalPersonNumber` scope.
+
+Section [2.1.2.1](#coordination-number-level) declares a claim for coordination number levels.
+This claim is seen as an informational claim, and needs to be explicitly requested using the
+`claims` request parameter. 
+
+| Claim | Description/comment | Reference |
+| :--- | :--- | :--- |
+| `https://id.oidc.se/claim/`<br />`personalIdentityNumber` | Swedish civic registration number. | This profile |
+| `https://id.oidc.se/claim/`<br />`coordinationNumber` | Swedish coordination number. | This profile |
+
+
+**Claims Parameter Equivalent:**
+
+```
+{
+  "userinfo" : {
+    "https://id.oidc.se/claim/personalIdentityNumber" : { "essential" : true },
+    "https://id.oidc.se/claim/coordinationNumber" : { "essential" : true }
+  },
+  "id_token" : {
+    "https://id.oidc.se/claim/personalIdentityNumber" : { "essential" : true },
+    "https://id.oidc.se/claim/coordinationNumber" : { "essential" : true }
+  }
+}
+```
+
+Since a Swedish personal identity number often is required to authenticate at a public
+Swedish organization these claims are delivered in the ID token and are marked as essential.
+The claims should also be delivered via the UserInfo endpoint.
 
 <a name="natural-person-organizational-identity"></a>
 ### 3.3. Natural Person Organizational Identity
 
 **Scope:** `https://id.oidc.se/scope/naturalPersonOrgId`
 
-**Description:** The “Natural Person Organizational Identity” scope provides basic organizational identity information about a
-person. The organizational identity does not necessarily imply that the subject has any particular relationship with or standing
-within the organization, but rather that this identity has been issued/provided by that organization for any particular reason
+**Description:** The “Natural Person Organizational Identity” scope requests basic organizational
+identity information claims about a person. The organizational identity does not necessarily imply
+that the subject has any particular relationship with or standing within the organization, but 
+rather that this identity has been issued/provided by that organization for any particular reason
 (employee, customer, consultant, etc.).
 
```

Comment:
Suggest removing "necessarily"

Also "any" and "particular" are kind of opposites. Would suggest "a particular purpose" or "any purpose"

Reply:
"Any particular relationsship" would translate to "Har någon specifik koppling" in Swedish, and that was what I wanted to say.